blogs/ecs-canary-deployments-pipeline/microservices/yelb-appserver/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM public.ecr.aws/bitnami/ruby:2.4.10
```

Comment:
q: do we not already have images in public ecr for yelb? do these assets diverge from yelb somehow?

Reply:
The yelb images have been sourced from Dockerhub and the official base images/yelb-images are not yet made available in public ECR. So we have done some testing with different versions of base images available in public ECR and started using the above declared.